packages/rocketchat-tokenpass/server/functions/lendTokenpassToken.js

```diff
@@ -0,0 +1,47 @@
+import moment from 'moment';
+
+let userAgent = 'Meteor';
+if (Meteor.release) { userAgent += `/${ Meteor.release }`; }
+
+RocketChat.lendTokenpassToken = function(lending, cb) {
```

Comment:
Meteor's server side HTTP API is synchronous, so there is no need for adding a callback style function here.

You would rather use it as synchronous or use Promises instead (when async is needed/desired)

Reply:
This got really bad :-/ I'll improve that.